openfecli/parameters/plan_network_options.py

```diff
@@ -193,6 +226,34 @@ def load_yaml_planner_options(path: Optional[str], context) -> PlanNetworkOption
     - `generate_minimal_redundant_network`
     - `generate_radial_network`
 
+
+Partial Charges
+===============
+
+Methods
+-------
+
+Supported partial charge method choices are:
+    - ``am1bcc``
+    - ``am1bccelf10`` (only possible if `off_toolkit_backend` in settings is set to `openeye`)
+    - ``nagl``
+    - ``espaloma`` (must have `espaloma charge installed`)
```

Comment:
```suggestion
    - ``espaloma`` (must have `espaloma_charge` installed)
```
(not sure that I'm actually correct here.)